cli/cmdlineparser.cpp

```diff
@@ -315,6 +315,9 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
 {
     mSettings.exename = Path::getCurrentExecutablePath(argv[0]);
 
+    bool mXmlOptionProvided = false;
```

Comment:
do not start local variable names with `m..` like this. The `m` stands for "member".